Assets/Scripts/ScriptableObjects/SceneDataBase.cs

```diff
@@ -0,0 +1,56 @@
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using System.Collections.Generic;
+
+[CreateAssetMenu(fileName = "SceneDataBase", menuName = "Scene Data/Database")]
+public class SceneDataBase : ScriptableObject
+{
+    public List<Level> levels = new List<Level>();
+    public string menu;
+    public int CurrentLevelIndex=0;
+
+    /*
+     * Levels
+     */
+
+    //Load a scene with a given index
+    public void LoadLevelWithIndex(int index)
+    {
+        if (index < levels.Count)
+        {
+            SceneManager.LoadScene(index);
+        }
+        //reset the index if we have no more levels
+        else CurrentLevelIndex =0;
```

Comment:
So we're changing the current level index if we cannot load a given scene id... But we stay on the current scene. This is weird

Reply:
Maybe in the "else" we want to simply call `LoadMainMenu()`?